src/utils/VectorFactory.ts

```diff
@@ -353,29 +365,32 @@ export function matrix16To6(matrix: any) {
 
 /*
  * This function receives an HTML5 canvas
- * along with a svg path for the clipping
- * mask and a divisor for the canvas scaling.
+ * along with a svg path for the clipping mask
  * It then draws the svg path onto the canvas.
  */
-export function clipCanvas(canvas: any, svgClipPath: any, divisor: any) {
-  divisor = divisor ? divisor : 1;
+export function clipCanvas(canvas: HTMLCanvasElement, svgClipPath: string, fillColor: string, scaleFactor: number) {
```

Comment:
It is probably not important, but I bet we could make this perform a bit better.

Reply:
Did you write this function? We just moved it over from the old site.

Reply:
Yeah, I wrote it a long time ago, and I don't remember how much time I spent thinking about optimization.  So it wouldn't hurt to give it a little thought since it has some loops that can run a many times, and also I haven't spent time thinking about performance of string operations (though I know split is generally as fast or faster than a regex).

Reply:
We are not seeing any performance issues there, so I suggest we wait with this.

Reply:
Agreed